data/core/rootview.lua

```diff
@@ -131,7 +136,18 @@ function RootView:on_mouse_pressed(button, x, y, clicks)
     end
   elseif not self.dragged_node then -- avoid sending on_mouse_pressed events when dragging tabs
     core.set_active_view(node.active_view)
-    return self.on_view_mouse_pressed(button, x, y, clicks) or node.active_view:on_mouse_pressed(button, x, y, clicks)
+    local in_gutter = node:in_tab_area_gutter(x, y)
+    if in_gutter and button == "left" then
+      if clicks == 2 then
```

Comment:
What do you think about `clicks % 2 == 0`?
This way one could open multiple new docs without having to wait for the click timeout to expire.

In the end, it's also fine like this.